config/system_factory_presets.go

```diff
@@ -18,12 +18,13 @@ import (
 func defaultProductionConfig() mutableNodeConfig {
 	cfg := emptyConfig()
 
-	cfg.SetUint32(PROTOCOL_VERSION, 1)
 	cfg.SetUint32(VIRTUAL_CHAIN_ID, 42)
 	cfg.SetUint32(GOSSIP_LISTEN_PORT, 4400)
 
-	cfg.SetDuration(MANAGEMENT_UPDATE_INTERVAL, 1*time.Minute)
+	cfg.SetDuration(MANAGEMENT_POLLING_INTERVAL, 1*time.Minute)
```

Comment:
let's reduce to ~ 10 sec for a smoother incerement

Reply:
done